docs/transaction-options.md

```diff
@@ -82,4 +82,10 @@ The following options are sepcific for each tx-type.
 
 ### SpendTx
 - `denomination` (default: `aettos`)
-    - You can specify the denomination of the `amount` that will be provided to the contract related transaction.
\ No newline at end of file
+    - You can specify the denomination of the `amount` that will be provided to the contract related transaction.
+
+## How to estimate gas?
+- As æpp developer, It is reasonable to estimate the gas consumption for a contract call using the dry-run feature of the node **once** and provide a specific offset (multiplied by 1.5 or 2) as default in the æpp to ensure that contract calls are mined. Depending on the logic of the contract the gas consumption of a specific contract call can vary and therefore you should monitor the gas consumption and increase the default for the respective contract call accordingly.
+- The default `gas` value of `25000` covers trivial contract calls but, The recommended way is estimating the gas using the dry-run feature instead of relying on the default value.
```

Comment:
```suggestion
- The default `gas` value of `25000` should cover all trivial contract calls. In case transactions start running out of gas you should proceed the way described above and estimate the required gas using the dry-run feature.
```